chocolatey/plugins/modules/win_chocolatey.ps1

```diff
@@ -649,8 +649,6 @@ Function Uninstall-ChocolateyPackage {
         $arguments.Add($timeout) > $null
     }
     if ($version) {
-        # Need to set allow-multiple to make sure choco doesn't uninstall all versions
-        $arguments.Add("--allow-multiple") > $null
```

Comment:
@vexx32 I'm concerned about that comment - does it now uninstall all versions if that is removed?

Reply:
Ignore this - it's answered over [here](https://github.com/chocolatey/chocolatey-ansible/issues/21#issuecomment-760286515).